wal/src/manager.rs

```diff
@@ -318,7 +318,7 @@ pub trait WalManager: Send + Sync + fmt::Debug + 'static {
     async fn scan(&self, ctx: &ScanContext, req: &ScanRequest) -> Result<BatchLogIteratorAdapter>;
 
     /// Get statistics
-    fn get_statistics(&self) -> Option<String> {
+    async fn get_statistics(&self) -> Option<String> {
         None
     }
```

Comment:
```suggestion
    async fn get_statistics(&self) -> Option<String>
```
Is it possible to make the method have no default implementation?

Reply:
It seems reasonable.

Reply:
Done.